core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/rules/auth/ExternalAuthorizationRule.scala

```diff
@@ -78,7 +80,7 @@ object ExternalAuthorizationRule {
   }
 
   final case class Settings(service: ExternalAuthorizationService,
-                            permittedGroupsLogic: GroupsLogic,
+                            groupsLogic: GroupsLogic,
```

Comment:
coutoPL comment migrated from old PR:

**_the same comment about naming and worsened context_**

Reply:
In this place, I think that the name "groupsLogic" is correct. ExternalAuthorizationRule supports all available logics, including negative and combined. So I think the name had to be changed to `groupsLogic` because of added types of GroupsLogic

Reply:
I'm not sure if I agree. I mean, your explanation makes sense, but in this setting field can be described as: "This is the logic that is supposed to calculate the available groups".

So, our external connector says: `user1 has groups A, B, C`. The logic says: `any but D`. It allows us to calculate available groups (or permitted groups)